src/terminal/terminal.c

```diff
@@ -1590,6 +1590,23 @@ static int __guac_terminal_send_key(guac_terminal* term, int keysym, int pressed
                 if (keysym == 0xFF54 || keysym == 0xFF99) return guac_terminal_send_string(term, "\x1B[B"); /* Down */
             }
 
+            if (term->mod_shift) 
+            {
```

Comment:
Please use the same style as the established code with respect to the `if` and the `{`. For example:

https://github.com/apache/guacamole-server/blob/491be8382acfc682a356284701a8535e4e73dd47/src/terminal/terminal.c#L1580

See: https://guacamole.apache.org/guac-style/#braces

Reply:
1. I resolved the formatting issue. And rewrote the commit message. 
2. Well if function key is pressed without Shift and with Shift. Both generate different key events. For the Ctrl+Function or Ctrl+Alt function or Ctrl+Alt+Shift +Function (This opens the menu and refocuses it to menu so function key hold with it, I am not sure the function key event will pass through.)

Reply:
Will add different functionality for the Alt and Ctrl hold and add to this PR branch.

Reply:
Added the different key combination support, in recent commit.